central/main.go

```diff
@@ -262,12 +262,6 @@ func main() {
 	log.Infof("Running StackRox Version: %s", pkgVersion.GetMainVersion())
 	log.Warn("The following permission resources have been replaced:\n" +
 		"	Access replaces AuthProvider, Group, Licenses, and User\n" +
-		"	Administration replaces AllComments, Config, DebugLogs, NetworkGraphConfig, ProbeUpload, ScannerBundle, ScannerDefinitions, SensorUpgradeConfig, and ServiceIdentity\n" +
-		"	Cluster also covers ClusterCVE\n" +
-		"	Compliance replaces ComplianceRuns\n" +
-		"	DeploymentExtension replaces Indicator, NetworkBaseline, ProcessWhitelist, and Risk\n" +
-		"	Integration replaces APIToken, BackupPlugins, ImageIntegration, Notifier, and SignatureIntegration\n" +
-		"	Image now also covers ImageComponent\n" +
 		"The following permission resources will be replaced in the upcoming versions:\n" +
 		"	Access will replace Role\n" +
```

Comment:
Not related to your stuff, but: this should have been moved with the latest PR by Yann, but apparently wasn't. Let's move it to the section above since it's actually replaced.

Reply:
Scratch that, I'll do it in my PR.